.pre-commit-config.yaml

```diff
@@ -36,12 +36,14 @@ repos:
     name: AWS CloudFormation Linter
     files: solutions/.*\.(yaml)$
     args: [--ignore-checks=W3002]
+    exclude: code/solutions/policy-as-code-with-guard/example_bucket_tests.yaml
```

Comment:
We could opt to use `.template` extension for CloudFormation files and avoid this sort of thing in general.

I tend to do this as a rule because CloudFormation YAML templates fail most generic YAML validators because of the non-standard tags (e.g. `!Ref`)

Reply:
Considering that existing example templates on this workshop use `.yaml` already, and if the exclusion path/pattern-driven logic works as expected for the use case, I think we should continue to use `.yaml` for consistency, unless we are considering to refactor this and other labs as well.

Reply:
Looking into it, we've mostly used .yaml throughout. I'll raise a separate issue to deal with it.